app/views/editions/secondary_nav_tabs/edit/editable/_guide.html.erb

```diff
@@ -2,10 +2,13 @@
 
 <%= render partial: "editions/secondary_nav_tabs/edit/editable/components/meta_tag_description", locals: { edition: @resource } %>
 
-<%= render "govuk_publishing_components/components/heading", {
-  text: "Chapters",
-  heading_level: 3,
-  font_size: "m",
+<%= render partial: "editions/secondary_nav_tabs/edit/editable/components/guide_chapter_list", locals: { edition: @resource } %>
+
+<%= render "govuk_publishing_components/components/button", {
```

Comment:
I think this button should only be rendered to users with the correct permissions.

Probably the `guide_add_new_chapter_page` page should also not be accessible to users without permissions (although the ticket does not explicitly state this).